src/components/Shop/OrderForm.tsx

```diff
@@ -9,7 +9,7 @@ function OrderForm() {
   const [discord, setDiscord] = useState('');
   const [packag, setPackage] = useState('');
   const [message, setMessage] = useState('');
-  const [budget, setBudget] = useState('');
+  const [budget, setBudget] = useState<number | string>('');
   const [deadline, setDeadline] = useState('');
```

Comment:
Hm... Nie dałoby się tych wszystkich błędów zebrać w jednej arajce, stanie, obiekcie, jakkolwiek by tego nie nazywać i tylko wrzucać do niej pojawiające się problemy? Coś na zasadzie:
`const errors = [];
if (budget <= 0) errors.push('Budget error')
...
return errors`
?
Bo teraz jeśli  będziesz chciał dodać kolejny błąd, to masz kolejny useState, kolejny warunek do ifa kawałek niżej, dużo zachodu dla czegoś, co można i powinno się znacznie uprościć. Zwróć zresztą uwagę, że potem masz dla każdego inputa:
`className={error ? 'input-error' : ''}`
Gdy mógłbyś mieć na przykład:
`className={errorsArray.includes('discord') ? 'input-error' : ''}`
Albo nawet metodę:
`className=checkIfHasErrors('discord')`
I metoda sprawdza w arajce errors czy błąd o danym kluczu, wartości, strukturze etc. występuje.
Innymi słowy spróbuj to zrobić bardziej generycznie, tak by nie musieć robić:
`if (
      clicked &&
      !emailError &&
      !discordError &&
      !packagError &&
      !messageError &&
      !budgetError &&
      !deadlineError
    )`
Bo to na dluższą metę się zemści.